code/modules/power/singularity/emitter.dm

```diff
@@ -1,6 +1,6 @@
 /obj/machinery/power/emitter
-	name = "Emitter"
-	desc = "A heavy duty industrial laser.\n<span class='notice'>Alt-click to rotate it clockwise.</span>"
+	name = "emitter"
+	desc = "A heavy-duty industrial laser used in containment fields and power generation.\n<span class='notice'>Alt-click to rotate it clockwise.</span>"

```

Comment:
It can, of course; I mainly put this here because in the context of the station it's used to contain giant black holes and to power the supermatter.

Also to kill blobs.